homeassistant/components/unifi/entity.py

```diff
@@ -151,25 +152,30 @@ async def remove_item(self, keys: set) -> None:
             await self.async_remove(force_remove=True)
 
     @callback
+    @abstractmethod
     def async_initiate_state(self) -> None:
         """Initiate entity state.
 
         Perform additional actions setting up platform entity child class state.
         """
+        raise NotImplementedError()
```

Comment:
There's no need to raise because it's not possible to instantiate an ABC class which has unimplemented `abstractmethod`s

Reply:
Ok. So similar examples online. Will fix

Reply:
Done